foreshadow/transformers/transformers.py

```diff
@@ -388,6 +388,15 @@ def fit_transform(self, X, *args, **kwargs):
                     out = _keep_columns_process(out, df, name)
             return out
 
+        def is_wrapped(self):
```

Comment:
I just realized, we shouldn't do a method because it will always exist and return True. When it doesnt, which is what we are trying to check, it will error out with an AttributeError which is not that graceful - likely better to use a utility function of foreshadow.utils to check and return True or False.

Reply:
Will patch